lib/cocina/models/purl.rb

```diff
@@ -3,7 +3,7 @@
 module Cocina
   module Models
     Purl = Types::String.constrained(
-      format: %r{^https://}i
+      format: %r{^https://}
```

Comment:
None of our PURL URLs should use capital letters. I will test this on sdr-infra.